vm-setup/roles/v1aX_integration_test/tasks/cleanup.yml

```diff
@@ -23,52 +23,70 @@
           (IMAGE_OS == "")
 
   - name: Check if cluster deprovisioning started.
-    shell: "kubectl get cluster -n {{ NAMESPACE }} -o json | jq -r '.items[] | .status.phase'"
+    k8s_info:
+      api_version: cluster.x-k8s.io/v1alpha3
```

Comment:
I think we also need to parameterize this since we would have two versions of these soon

Reply:
Done